File: Lab4/t2.cpp
#include <stdio.h>
int main(){
    int a, b, c;
    scanf("%d%d%d", &a, &b, &c);
    puts("Please input three integers: ");
    printf("Among %d, %d and %d, %d is the greatest.\n", a, b, c, (a > b ? a : b) > c ? (a > b ? a : b) : c);
    return 0;
}

File: Lab4/t2.java
import java.util.Scanner;
public class t2{
    public static void main(String[] args) {
        int a, b, c;
        Scanner scan = new Scanner(System.in);
        System.out.println("Please input three integers: ");
        a = scan.nextInt();
        b = scan.nextInt();
        c = scan.nextInt();
        scan.close();
        System.out.printf("Among %d, %d and %d, %d is the greatest.\n", a, b, c, (a > b ? a : b) > c ? (a > b ? a : b) : c);
    }
}